Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -13073,7 +13086,28 @@
 		};
 /* End XCConfigurationList section */
 
+/* Begin XCRemoteSwiftPackageReference section */
+		3C6C7FD92B457381006F5CBC /* XCRemoteSwiftPackageReference "opentelemetry-swift" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/open-telemetry/opentelemetry-swift.git";
+			requirement = {
+				kind = upToNextMajorVersion;
+				minimumVersion = 1.9.0;
+			};
+		};
+/* End XCRemoteSwiftPackageReference section */
+
 /* Begin XCSwiftPackageProductDependency section */
+		3C6C7FDA2B45738C006F5CBC /* OpenTelemetryApi */ = {
+			isa = XCSwiftPackageProductDependency;
+			package = 3C6C7FD92B457381006F5CBC /* XCRemoteSwiftPackageReference "opentelemetry-swift" */;
+			productName = OpenTelemetryApi;
+		};
+		3C6C7FDC2B457392006F5CBC /* OpenTelemetryApi */ = {
+			isa = XCSwiftPackageProductDependency;
+			package = 3C6C7FD92B457381006F5CBC /* XCRemoteSwiftPackageReference "opentelemetry-swift" */;
+			productName = OpenTelemetryApi;
+		};
```

Comment:
duplicated?

Reply:
nope - same as above.

One for iOS and another for tvOS.